### PR TITLE
George

### PR DESCRIPTION
**Pull request by georgeSoton**

### CODE REVIEW

**Comment by tblah** [comment]
(would approve if I could see the button)

File: first_models.tex [hunk 1]
```diff
@@ -1,14 +1,164 @@
...
+\label{sec:bag_of_words}
+From the examination of the data in Section \ref{sec:word_frequency}, a bag-of-words classifier seems plausible. Through a direct comparison of the frequencies with which each author uses the words in a given sentence, a prediction may be formed.
 
-  \subsection{Forest}
-  \label{sec:forest}
+Let $\phi_w^a$ be the number of occurrences of some word $w$ in the corpus of sentences attributed to author $a$. The estimated probability of an author's usage of a word, $w^*$, is given by:
 
-  \subsection{DNN}
-  \label{sec:dnn}
+\begin{equation*}
```
Comment (superdyoll) [inline first_models.tex:10]: Do these equations need to be numbered or lettered?

Comment (georgeSoton) [reply first_models.tex:10]: I don't think it serves any purpose and just clutters it.

File: extensions.tex [hunk 2]
```diff
@@ -40,21 +41,50 @@ \subsection{Beta Distribution Bag-of-words}
...
+  \item The max and min log probability of all words
+  \item The standard deviation of log probabilities of all words
+  \item The number of "missed" words in the sentence \footnote{"Missed" words are those not observed as having been used by an author in the training set. Whilst they are still assigned a probability as in Section \ref{sec:bow_beta}, their record as having been missed is retained}
+  \end{itemize}
+  
+ Additionally, the sentence length is included based on the observations in Section \ref{sec:sentence_lengths}. This provides a 16 neuron input-layer.
+ 
+ The confusion matrix, accuracy and loss for this model may be seen in Table \ref{tab:bow_nn_res}. The results presented utilise a perceptron with no hidden layers; adding hidden layers does not improve performance. Noticeable overfitting is observed with this architecture. Its training loss is $<0.2$, and its training accuracy $>90%$. Attempts to overcome this such as compressing the model through a 2-neuron layer and increasing batch sizes were not successful.
```
Comment (Adimote) [inline extensions.tex]: compile error: missing an escape on the percent sign `$>90%$` should be  `$>90\%$`

**Review by superdyoll — APPROVED** [review]

**Pull request merged**